waiter/specs/k8s-default-pod.edn

```diff
@@ -0,0 +1,38 @@
+{:kind "ReplicaSet"
+ :apiVersion "extensions/v1beta1"
+ :metadata {:name #waiter/param :k8s-name
+            :labels {:app #waiter/param :k8s-name
+                     :managed-by "waiter"}
```

Comment:
That's implied by the namespace here.

Reply:
All of the examples in the Kubernetes docs that I've seen use the `app` label for the application name. E.g.: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#more-practical-use-cases

Reply:
Do we want to add it anyway in case we choose some other namespace mechanism later?

Reply:
My impression is that adding extra labels to a Kubernetes object is pretty cheap, so I guess there's no harm in adding it.

Reply:
sgtm :)

Reply:
OK, thinking about this more, this seems very redundant for something that we don't anticipate using anytime soon. Since it's a fairly lightweight thing, it will be simple to add it in the future if we decide to stop assuming the namespace⇔run-as-user correspondence.